### PR TITLE
[ready to publish] Spanish - 3 WCAG resources

### PR DESCRIPTION
**Pull request by cmuncharaz**
Translation of the "WCAG 2.1 At a Glance" document into Spanish.

### CODE REVIEW

**Comment by slhenry** [comment]
Hi Carlos. This is what is stopping the preview:
> {% include box.html type="start" h="2" title="Resumen="full" %}

should be:
> {% include box.html type="start" h="2" title="Resumen" class="full" %}

 :-)

**Comment by cmunch** [comment]
Great catch, @slhenry! I was about to write you asking about this issue. Many thanks!

**Comment by yatil** [comment]
There still seems to be an issue. I’ll check it out. (quite literally) :-)

**Comment by cmunch** [comment]
Yep. I was checking the log but could not find any error, only a message informing about a failure to build the site. I am clueless about what could be wrong.

**Comment by cmunch** [comment]
Awesome. Thank you @yatil!

**Comment by yatil** [comment]
@cmunch I wished the reporting was more explicit. It is locally on my machine for some reason. I fixed the typo. :-)

**Comment by slhenry** [comment]
Hi @rumoroso & @espinr

These pages are ready for your review:

1. Introducción a las Pautas de Accesibilidad para el Contenido Web (WCAG)
Preview: https://deploy-preview-38--wai-intro-wcag.netlify.com/standards-guidelines/wcag/es

2. WCAG 2.1 de un vistazo
Preview: https://deploy-preview-38--wai-intro-wcag.netlify.com/standards-guidelines/wcag/glance/es

2. Novedades en WCAG 2.1
Preview: https://deploy-preview-38--wai-intro-wcag.netlify.com/standards-guidelines/wcag/new-in-21/es

(We'll do separate pull requests for each resource in the future. ;-)

File: new-in-21.es.md [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+<a href="https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html">Comprender el Espaciado de Texto</a>
+
+### 1.4.13 Contenido en Puntero Flotante o en Foco (AA)
+
+<blockquote class="sc">
+  <p>Si mover y alejar el puntero o el foco del teclado con respecto a un elemento resulta en contenido adicional que se muestra y después se oculta, entonces lo siguiente debe ser cierto:</p>
+  <dl>
+    <dt>Descartable</dt>
+    <dd>Debe existir un <a href="https://www.w3.org/TR/WCAG21/#dfn-mechanism" data-link-type="dfn">mecanismo</a> para descartar el contenido adicional sin mover el puntero o el foco del teclado, salvo que el contenido adicional informe acerca de un <a href="https://www.w3.org/TR/WCAG21/#dfn-input-error" data-link-type="dfn">error de entrada</a> o no oculte ni sustituya otro contenido;</dd>
+    <dt>Sobrevolable</dt>
```
Comment (espinr) [inline new-in-21.es.md]: Proposal: '[Superponible](https://dle.rae.es/srv/search?m=30&w=superponible)'?

File: new-in-21.es.md [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+<p><a href="https://www.w3.org/WAI/WCAG21/Understanding/concurrent-input-mechanisms.html">Comprender los Mecanismos de Entrada Concurrentes</a></p>
+
+## Pauta 4.1 Compatible
+
+Maximice la compatibilidad con agentes de usuario actuales y futuros, incluyendo las ayudas técnicas.
+
+### 4.1.3 Mensajes de Estado (AA)
+
+<blockquote class="sc">
+  <p>En el contenido implementado usando lenguajes de marcado, los <a href="https://www.w3.org/TR/WCAG21/#dfn-status-messages" data-link-type="dfn">mensajes de estado</a> se pueden <a href="https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable" data-link-type="dfn">determinado por software</a> a través de <a href="https://www.w3.org/TR/WCAG21/#dfn-role" data-link-type="dfn">roles</a> o propiedades tales que puedan ser presentados al usuario de <a href="https://www.w3.org/TR/WCAG21/#dfn-assistive-technologies" data-link-type="dfn">tecnologías de apoyo</a> sin necesidad de recibir el foco.</p>
```
Comment (espinr) [inline new-in-21.es.md]: s/determinado/determinar

File: new-in-21.es.md [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+    <li><p><span class="issue">Solución:</span><span><q>Cuando añado una reunión en mi calendario, oigo una confirmación.</q></span></p></li>
+  </ul>
+</div>
+<p><a href="https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html">Comprender los Mensajes de Estado</a></p>
+
+## Sobre las citas de las personas
+
+Los roles de personas enlazan con las [[Historias de usuarios web]](/people-use-web/user-stories/). Dicha página contiene otras personas con diferentes discapacidades. Se podrían añadir más en el futuro.
+
+Una vez estas personas se hayan revisado suficientemente, planeamos añadirlas a los documentos de Comprender.
```
Comment (espinr) [inline new-in-21.es.md]: documentos 'Comprender'?

File: index.es.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+Los materiales de apoyo de las WCAG 2 comprenden:
+
+-   [**Cómo cumplir con las WCAG 2**: Una guía de referencia rápida y personalizable que incluye los requisitos (criterios de conformidad) y técnicas de las Pautas de Accesibilidad para el Contenido Web](http://www.w3.org/WAI/WCAG21/quickref/) que es, en esencia, [la lista de control de WCAG 2](http://www.w3.org/WAI/WCAG21/quickref/). La mayoría de la gente utiliza esta guía de referencia rápida como herramienta principal para trabajar con las WCAG.
+-   **Técnicas para las WCAG 2** ([Técnicas 2.1](https://www.w3.org/WAI/WCAG21/Techniques/), [Técnicas 2.0](https://www.w3.org/TR/WCAG20-TECHS/)) proporcionan información específica para desarrollar contenido accesible, como por ejemplo fragmentos de código HTML. Las técnicas son “informativas”, lo cual quiere decir que no tiene por qué usarlas. La base para determinar la conformidad con WCAG 2 reside en los *criterios de conformidad* del texto WCAG 2 estándar y no en las técnicas. Lea más sobre las [Técnicas en el FAQ](/standards-guidelines/wcag/faq/#techs).
```
[the remark this answers is not in the document]
Comment (cmuncharaz) [reply index.es.md]: Love you vim style change proposals, @espinr :laughing:

File: new-in-21.es.md [hunk 5]
```diff
@@ -0,0 +1,430 @@
...
+  <ul>
+    <li>El campo de entrada sirve a un propósito identificado en los <a href="https://www.w3.org/TR/WCAG21/#input-purposes">propósitos de entrada de los componentes de la interfaz de usuario</a>; y</li>
+    <li>El contenido se implementa usando tecnologías con soporte para identificar el significado esperado de los datos de entrada en formularios.</li>
+  </ul>
+</blockquote>
+<p class="persona">Un <a href="https://www.w3.org/WAI/people-use-web/user-stories/#supermarketassistant">auxiliar de supermercado</a> con dislexia y discalculia:</p>
+<div class="quotes">
+  <ul>
+    <li><p><span class="issue">Problema:</span><span><q>Mi dirección es muy complicada. Tiene muchos números y palabras largas. Me resulta difícil teclearlo todo sin cometer errores.</q></span></p></li>
+    <li><p><span class="issue">Solución:</span><span><q>Me encantan los sitios web que automáticamente rellenan todo eso por mí. De esta forma, no tengo que esforzarme tanto para asegurar que los números y las palabras están bien.</q><br/>
```
[the remark this answers is not in the document]
Comment (cmuncharaz) [reply new-in-21.es.md]: Agree. At first I was using "Funciona bien" but was not really happy and eventually changed it for the simpler "Solución". Having someone else happy with "Funciona bien" makes it easier to go with it.

File: new-in-21.es.md (hunk 1 above)
Comment (cmuncharaz) [reply new-in-21.es.md]: Great, just learned a new word!

**Comment by slhenry** [comment]
Thanks, Carlos!

Let me know when you're done and I'll skim it and queue it up for publication tomorrow.

**Comment by cmuncharaz** [comment]
It's done, @slhenry!

**Comment by slhenry** [comment]
@yatil notice there are 3 files in this PR

**Pull request merged**